packages/app/src/components/Common/ImageCropModal.tsx

```diff
@@ -93,31 +95,68 @@ const ImageCropModal: FC<Props> = (props: Props) => {
     }
   };
 
-  const crop = async() => {
-    // crop immages
+  // Convert base64 Image to blob
+  const convertBase64ToBlob = async(base64Image: string) => {
+    const base64Response = await fetch(base64Image);
+    const blob = await base64Response.blob();
+    return blob;
+  };
```

Comment:
Just checking but, did it work properly?

Reply:
Yes, It runs smoothly without many lines of code